lib/config.js

```diff
@@ -21,17 +21,11 @@ cfg.storageCost = {
   revocation: 1
 };
 
-// create dev meter usage reporter client (must be overridden in deployments)
+// create dev application identity for edv (must be overridden in deployments)
 // ...and `ensureConfigOverride` has already been set via
-// `bedrock-meter-usage-reporter` so it doesn't have to be set here
-config['meter-usage-reporter'].clients.edv = {
-  id: 'did:key:z6MkgwieJDAgdUQyD17L6TsGJon57jt3yxZrtw7rg3tJ5orH',
-  keyPair: {
-    id: 'did:key:z6MkgwieJDAgdUQyD17L6TsGJon57jt3yxZrtw7rg3tJ5orH#' +
-      'z6MkgwieJDAgdUQyD17L6TsGJon57jt3yxZrtw7rg3tJ5orH',
-    type: 'Ed25519VerificationKey2020',
-    publicKeyMultibase: 'z6MkgwieJDAgdUQyD17L6TsGJon57jt3yxZrtw7rg3tJ5orH',
-    privateKeyMultibase: 'zrv4SV17sr6L2Ymyyu3Uj7cvJm3qvmMKbzW1wwJkpuGiQj5' +
-      'XqQeF5nmtwoP86VH7zzT2v4uS921gJzbztbdbCakwd8D'
-  }
+// `bedrock-app-identity` so it doesn't have to be set here
+config['app-identity'].seeds.services.edv = {
+  id: 'did:key:z6MkhNyDoLpNcPv5grXoJSJVJjvApd46JU5nPL6cwi88caYW',
+  seedMultibase: 'z1AgcCz4zGY5P3covUxqpaGTVs6U12H5aWH1FdyVABCwzkw',
+  serviceType: 'edv'
```

Comment:
Do we need `serviceType` here?

Reply:
Yes, the name of the key may not equal the `serviceType` in some cases.